CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing to Jenkinsfile Runner
+
+This page provides information about contributing code to the Jenkinsfile Runner codebase.
+
+## Getting started
+
+1. Fork the repository on GitHub
+2. Clone the forked repository to your machine
+3. Install the development tools. In order to develop Jenkinsfile Runner, you need the following tools:
+  * Java Development Kit (JDK) 8.
+     - We usually use [OpenJDK](http://openjdk.java.net/) but you can use other JDKs as well.
+  * Maven 3.5.3 or above.
```

Comment:
Better to recommend 2.5.4, but not a big difference

Reply:
Isn't that one too old? I don't think it hurts to recommend a newer one.

Reply:
Sorry, 3.5.4

```suggestion
  * Maven 3.5.4 or above.
```